de/news/_posts/2024-12-25-ruby-3-4-0-released.md

```diff
@@ -158,6 +158,22 @@ Hinweis: Es werden nur erwähnenswerte Updates der Standardbibliotheken aufgefü
 
   - Diese Bibliothek wurde aus dem Ruby-Repository nach [ruby/net-http-sspi] extrahiert. [[Feature #20775]]
 
+Das folgende mitinstallierten Gem wird von Standard-Gems gefördert.
```

Comment:
https://github.com/ruby/www.ruby-lang.org/blob/3b8ae042e2869165c60e29563ae6c8d4beb25c77/de/news/_posts/2023-12-25-ruby-3-3-0-released.md?plain=1#L248-L250

Reply:
please change to:

```
Die folgenden Bundled-Gems sind ab nun teil der Standard-Gems:
```